lua/lspconfig/configs/ccls.lua

```diff
@@ -7,7 +7,7 @@ local function switch_source_header(bufnr)
     vim.notify('method textdocument/switchsourceheader is not supported by any servers active on the current buffer')
   end
   local params = vim.lsp.util.make_text_document_params(bufnr)
-  client.request('textdocument/switchsourceheader', params, function(err, result)
+  client.request('textDocument/switchsourceheader', params, function(err, result)
```

Comment:
The correct spelling is `textDocument/switchSourceHeader` and the message above needs to be fixed as well.

Reply:
will fix on refactor PR there lots of duplicate codes on clangd.

Reply:
`switchsourceheader` is correct? in clangd is `switchSourceHeader`

Reply:
`textDocument/switchSourceHeader`, the same as clangd :)

I had used the function copied from clangd for a while https://github.com/MaskRay/Config/blob/1f431c999079e900bbe7958a00578cba4c4b3b67/home/.config/nvim/lua/my/util.lua#L41  until I attempted to upstream it...